include/eve/module/core/regular/abs.hpp

```diff
@@ -35,50 +47,44 @@ namespace eve
 //!
 //!   **Parameters**
 //!
-//!     * `x` :  [real argument](@ref eve::value).
+//!     * `x` :  [SIMD or scalar arithmetic value](@ref eve::value).
 //!
 //!    **Return value**
 //!
-//!     *  value containing the [elementwise](@ref glossary_elementwise)
-//!        absolute value of `x` if it is representable in this type.
-//!
-//!        More specifically, for signed integers : the absolute value of eve::valmin
-//!        is not representable and the result is undefined.
+//!    The [elementwise](@ref glossary_elementwise) absolute value of `x`, if it is representable.
+//!    More specifically, for signed integers : the absolute value of eve::valmin is not representable and
+//!    the result is undefined.
 //!
-//!  @warning
-//!        abs is also a standard library function name and there possibly
-//!        exists a C macro version which may be called instead of the EVE version.
-//!        To avoid confusion, use the eve::abs notation.
+//!   @warning
+//!   `abs` is also a standard library function name and there possibly
+//!   exists a C macro version which may be called instead of the EVE version.<br/>
+//!   To avoid confusion, use the eve::abs notation.
 //!
-//!  @groupheader{Example}
+//!   @groupheader{Example}
 //!
-//!  @godbolt{doc/core/regular/abs.cpp}
+//!   @godbolt{doc/core/regular/abs.cpp}
 //!
-//!  @groupheader{Semantic Modifiers}
+//!   @groupheader{Semantic Modifiers}
```

Comment:
should this just be one doc? upp to you, I'm just asking

Reply:
I was thinking about it, I'll try something - i.e having a single doc .cpp and use snippet inside doxygen to cut it in pieces